src/sentry/models/event.py

```diff
@@ -199,7 +199,7 @@ def as_dict(self):
 
     @property
     def size(self):
```

Comment:
we could also just make this json dump/load if we really wanted to -- id like to memoize it if we were to do more, but it'd need to then react to Event.data changes which complicates things

Reply:
Yeah, I'm not even entirely sure what "size" here is supposed to represent since this currently doesn't correlate to storage size or anything since it's not the Node size.

So any change here is just arbitrary and shouldn't impact anything that we rely on.